src/main/java/core/basesyntax/models/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.models;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(int radius, String color) {
+        this.radius = radius > 0 ? radius : 1;

```

Comment:
There shouldn't be any logic inside the constructor. The constructor should only create a new instance of the class